crates/primitives/starknet/Cargo.toml

```diff
@@ -25,7 +25,9 @@ starknet-ff = { workspace = true, default-features = false, features = [
 	"alloc",
 ] }
 poseidon_hash = { workspace = true, default-features = false }
-blockifier = { workspace = true, default-features = false, features = ["testing"]}
+blockifier = { workspace = true, default-features = false, features = [
+	"testing",
```

Comment:
it looks like we don't all have the same formatter, not sure how to standardize this though, may investigate

Reply:
Better TOML
Id: bungcip.better-toml
Description: Better TOML Language support
Version: 0.3.2
Publisher: bungcip
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=bungcip.better-toml

Reply:
I have this VS code extension enabled. It's probably the one messing with TOMLs